backend/app/controllers/spree/admin/taxons_controller.rb

```diff
@@ -3,6 +3,7 @@
 module Spree
   module Admin
     class TaxonsController < Spree::Admin::BaseController
+      rescue_from ActiveRecord::RecordNotFound, with: :resource_not_found
```

Comment:
@ikraamg I believe the suggestion @aldesantis was making is to modify the calls to `rescue_from` (like this one) to avoid triggering the passing of the exception object by calling it like this:

```suggestion
      rescue_from ActiveRecord::RecordNotFound do
        resource_not_found
      end
```

This way we do not need the unused optional exception argument.

Reply:
@jarednorman thanks for clarifying the suggestion, I completely misunderstood! 
This worked great @aldesantis ! I agree that the unused argument was an oddity.